### PR TITLE
Updates to native lazy-loading guide

### PR DESCRIPTION
**Pull request by addyosmani**
Changes proposed in this pull request:

- Re-focus this guide around native image lazy-loading (we'll cover iframes in more detail separately)
- Expand guidance to better capture nuance about the Chromium implementation
- Revise some of the FAQs and best practices 

FYIs: @scott-little @domfarolino

There are still some changes I would like to land here covering CMS, but sharing progress so far.

### CODE REVIEW

File: src/site/content/en/fast/native-lazy-loading/index.md [hunk 1]
```diff
@@ -120,20 +113,26 @@ the meantime, you will need to override the effective connection type of the bro
...
+
+```html
+<img src="image.png" loading="lazy" alt="…" style="height:200px; width:200px;">
+```
+
+The best practice of setting dimensions applies to `<img>` tags regardless of whether or not they are being loaded lazily. With lazy-loading, this can become more relevant. Setting `width` and `height` on images in modern browsers also allows browsers to infer their instrinsic size.
+
+Images will still lazy-load if dimensions are not included, but [specifying them decreases the chance of layout shift](https://www.youtube.com/watch?v=4-d_SoCHeWE). If you are unable to include dimensions for your images, lazy-loading them can be a trade-off between saving network resources and potentially being more at risk of layout shift. 
+
+While native lazy-loading in Chromium is implemented in a way such that images are likely to be loaded once they are visible, it should be kept in mind, there is a slightly greater chance of them not being loaded yet. In this case, case missing `width` and `height` attributes on such images increase their impact on Cumulative Layout Shift.
```
Comment (housseindjirdeh) [inline src/site/content/en/fast/native-lazy-loading/index.md:135]:
```suggestion
While native lazy-loading in Chromium is implemented in a way such that images are likely to be loaded once they are visible, it should be kept in mind, there is a slightly greater chance of them not being loaded yet. In this case, missing `width` and `height` attributes on such images increase their impact on Cumulative Layout Shift.
```

File: src/site/content/en/fast/native-lazy-loading/index.md [hunk 2]
```diff
@@ -150,40 +149,53 @@ Images that are defined using the `<picture>` element can also be lazy-loaded:
...
 
-### Are there plans to expand this feature?
+### Are there plans to automatically lazy-load images in Chrome?
 
-There are plans to change the default lazy-loading behavior of the browser to automatically
-lazy-load any images or iframes that are well suited to being deferred if [Lite
-mode](https://blog.chromium.org/2019/04/data-saver-is-now-lite-mode.html) is enabled on Chrome for
-Android.
+Chromium already automatically
+lazy-load any images that are well suited to being deferred if [Lite
```
Comment (housseindjirdeh) [inline src/site/content/en/fast/native-lazy-loading/index.md:186]:
```suggestion
lazy-loads any images that are well suited to being deferred if [Lite
```

File: src/site/content/en/fast/native-lazy-loading/index.md [hunk 3]
```diff
@@ -132,7 +130,7 @@ The best practice of setting dimensions applies to `<img>` tags regardless of wh
 
 Images will still lazy-load if dimensions are not included, but [specifying them decreases the chance of layout shift](https://www.youtube.com/watch?v=4-d_SoCHeWE). If you are unable to include dimensions for your images, lazy-loading them can be a trade-off between saving network resources and potentially being more at risk of layout shift. 
 
-While native lazy-loading in Chromium is implemented in a way such that images are likely to be loaded once they are visible, it should be kept in mind, there is a slightly greater chance of them not being loaded yet. In this case, case missing `width` and `height` attributes on such images increase their impact on Cumulative Layout Shift.
+While native lazy-loading in Chromium is implemented in a way such that images are likely to be loaded once they are visible, it should be kept in mind, there is a slightly greater chance of them not being loaded yet. In this case, missing `width` and `height` attributes on such images increase their impact on Cumulative Layout Shift.
```
Comment (domfarolino) [inline src/site/content/en/fast/native-lazy-loading/index.md:133]: This isn't related to this PR, but "there is a slightly greater chance of them not being loaded yet" makes it sound like there is a greater chance that they won't be loaded _compared to_ them being loaded. Is that true? I feel like the intention of this sentence is to say that "there is still a small chance that they might not be loaded yet [...]". Thoughts?

**Comment by domfarolino** [comment]
> There are still some changes I would like to land here covering CMS, but sharing progress so far.

I look forward to these, but for now, the changes look good.

File: src/site/content/en/fast/native-lazy-loading/index.md (hunk 3 above)
Comment (addyosmani) [reply src/site/content/en/fast/native-lazy-loading/index.md:133]: The wording here could certainly be improved. I actually like your proposed version of this sentence. I'm going to go ahead and integrate this into the sentence as it's worded more clearly. Thanks, Dom!

**Comment by addyosmani** [comment]
> I look forward to these, but for now, the changes look good.

Thank you, Dom! I have also just got word from Scott that he is comfortable with the numbers mentioned in the PR. Going to go ahead and land this.

Appreciate your reviews here, everyone!

**Comment by konojunya** [comment]
ref: #3469

**Pull request merged**